### PR TITLE
Blog post about DockerCon sponsorship

### PR DESCRIPTION
**Pull request by tbrandles**

### CODE REVIEW

File: crate/pages/blog/dockercon-gold-sponsor.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+category: news
+og_image: https://cdn.crate.io/media/1503/crate-dockercon.png\
+
+<img src="https://cdn.crate.io/media/1503/crate-dockercon-horizontal.png" width="600" height="98" />
+
+We are proud to announce that Crate.IO is a gold sponsor for DockerCon 2015. 
+
+After attending DockerCon in Amsterdam last year we fell in love with Docker and wanted get involved with the community.  <a href="{% url 'http://blog.docker.com/2015/01/dockercon-eu-introducing-docker-swarm/' %}">Docker Swarm</a> and <a href="{% url 'http://blog.docker.com/2015/01/dockercon-eu-introducing-docker-compose/' %}">Docker Compose</a> made their debut in at DockerCon last year (we have found both tools to be extremely <a href="{% url '/blog/deploying-crate-with-docker-machine-swarm.html' %}">helpful</a>) and we can't wait to see what announcements will be made this year.
+
+Join the Crate team at the <a href="{% url 'https://www.eventbrite.com/e/dockercon-sf-hackathon-tickets-15613226581' %}">DockerCon Hackathon</a>, come to one of our talks during the conference, or visit us at our booth.
```
Comment (chaudum) [inline crate/pages/blog/dockercon-gold-sponsor.html]: {% url '...' %} must only be used for internal urls

File: crate/pages/blog/dockercon-gold-sponsor.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+title: Dockercon gold sponsor
+author: Tyler Randles
+description: 
+created: 2015/03/24 12:00:00
+post_name: dockercon-gold-sponsor
+status: publish
+post_type: post
+tags: docker, dockercon, conference
+category: news
+og_image: https://cdn.crate.io/media/1503/crate-dockercon.png\
```
Comment (chaudum) [inline crate/pages/blog/dockercon-gold-sponsor.html]: remove trailing \

File: crate/pages/blog/dockercon-gold-sponsor.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+<img src="https://cdn.crate.io/media/1503/crate-dockercon-horizontal.png" width="600" height="98" />
```
Comment (chaudum) [inline crate/pages/blog/dockercon-gold-sponsor.html]: use `title` markdown syntax
or `{% media '/media/1503/crate-dockercon-horizontal.png' %}`

File: crate/pages/blog/dockercon-gold-sponsor.html [hunk 4]
```diff
@@ -0,0 +1,18 @@
+title: Dockercon gold sponsor
```
Comment (chaudum) [inline crate/pages/blog/dockercon-gold-sponsor.html]: please use title case

File: crate/pages/blog/dockercon-gold-sponsor.html [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+author: Tyler Randles
+description:
```
Comment (chaudum) [inline crate/pages/blog/dockercon-gold-sponsor.html]: add description

File: crate/pages/blog/dockercon-gold-sponsor.html [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+post_name: dockercon-gold-sponsor
+status: publish
+post_type: post
+tags: docker, dockercon, conference
+category: news
+og_image: https://cdn.crate.io/media/1503/crate-dockercon.png\
+
+<img src="https://cdn.crate.io/media/1503/crate-dockercon-horizontal.png" width="600" height="98" />
+
+We are proud to announce that Crate.IO is a gold sponsor for DockerCon 2015.
```
Comment (chaudum) [inline crate/pages/blog/dockercon-gold-sponsor.html]: link to DockerCon website?

**Comment by tbrandles** [comment]
ACK

**Pull request merged**